ui/src/hooks/useFetchStatuses.tsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useState } from "react"
+
+export interface StackStatus {
+  Name: string
+  Error: string
+  Revision: string
+  RepoURL: string
+}
+
+const devData: StackStatus[] = [
+  {
+    Name: "Project Alpha",
+    Error: "",
+    Revision: "v1.0.1",
+    RepoURL: "https://github.com/user/project-alpha"
+  },
+  {
+    Name: "Project Beta",
+    Error: "Failed to build",
+    Revision: "v2.3.4",
+    RepoURL: "https://github.com/user/project-beta"
+  },
+  {
+    Name: "Project Gamma",
+    Error: "",
+    Revision: "v0.9.8",
+    RepoURL: "https://github.com/user/project-gamma"
+  }
+]
+
```

Comment:
Should be in a separate file from production code

Reply:
Put into `dummyStackStatuses.json`